sourcemash/frontend/templates/saved.tpl

```diff
@@ -11,10 +11,10 @@
 <div id="subscribe-modal" class="modal">
   <div class="modal-content">
     <h4>Interested?</h4>
-    <p>You upvoted "<span id="unsubscribed-item-title"></span>" even though you're not subscribed to <i><span id="unsubscribed-feed-title"></span></i>. Would you like to add this feed to your subscriptions?<p>
+    <p>You saved or upvoted "<span id="unsubscribed-item-title"></span>" even though you're not subscribed to <i><span id="unsubscribed-feed-title"></span></i>. Would you like to add this feed to your subscriptions?<p>
```

Comment:
Do we need one of these on the saved tpl? Would't they already have clicked save on the items they see here?